apps/src/templates/projects/projectsRedux.js

```diff
@@ -192,10 +194,33 @@ function personalProjectsList(state = initialPersonalProjectsList, action) {
         ...updatedNotEditing[projectNoLongerBeingRenamedIndex],
         isEditing: false,
       };
+    return {
+      ...state,
+      projects: updatedNotEditing,
+    };
+    case SAVE_SUCCESS:
+      var recentlySavedProjectId = action.projectId;
```

Comment:
style nit: instantiate variables on 202, 204, 206, and 208 as `const` if they won't be mutated. also, this is obviously personal preference, but you can remove the extra newlines between the variable assignments on these lines as well

Reply:
This angers the linter. I get `201:5  error  Unexpected lexical declaration in case block  no-case-declarations`.

Reply:
oh, weird! i suppose that makes sense though. you can also use `let` to make sure the [scope is limited](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) to this block, although the linter might yell about that, too